iib/web/models.py

```diff
@@ -997,6 +1030,14 @@ def from_json(cls, kwargs, batch=None):
         """
         request_kwargs = deepcopy(kwargs)
 
+        for key in ('build_tags',):
```

Comment:
I don't think you need this validation here right? Since you are calling the parent class `_from_json`?

Reply:
but there's no validation for build_tags there

Reply:
Do you need braces around the build_tags?

Reply:
I will add it to the Request._from_json if you want to use it for RequestAdd, RequestRm, RequestCreateEmptyImage,

Reply:
Same as above, moving this check from here to _from_json will help you get rid of this code duplication.

Reply:
Simply pass `build_tags` in the call on line 1050 while calling `cls._from_json` under `additional_optional_params ` and that would do the trick.

Reply:
Same can be done for all three classes `RequestAdd`, `RequestRm` and `RequestMergeIndex`